chain-impl-mockchain/src/transaction/transfer.rs

```diff
@@ -6,7 +6,7 @@ use chain_core::mempack::{ReadBuf, ReadError, Readable};
 use chain_core::property;
 use chain_crypto::PublicKey;
 
-const INPUT_PTR_SIZE: usize = 32;
+pub const INPUT_PTR_SIZE: usize = 32;
```

Comment:
why do you need this public?

Reply:
Because we need to generate an input pointer. This probably should be public in the first place, because it's a part of `Input` and `AccountIdentifier` public APIs